- незначительно скорректировал AdController

Garik-create opened the pull request:
- добавил поля для связи сущностей в AdEntity, UserEntity
- скорректировал тип идентификатора в AdRepository, UserRepository
- добавил CRUD-методы в AdService, AdServiceImpl (кроме updateComment)
- добавил запросы к БД в CommentRepository
- добавил маппер ResponseWrapperCommentMapper и класс ResponseWrapperCommentMapperImpl
- добавил структуру таблиц проекта в WiKi проекта

Garik-create committed:
- незначительно скорректировал AdController
- добавил поля для связи сущностей в AdEntity, UserEntity
- скорректировал тип идентификатора в AdRepository, UserRepository
- добавил CRUD-методы в AdService, AdServiceImpl (кроме updateComment)
- добавил запросы к БД в CommentRepository
- добавил маппер ResponseWrapperCommentMapper и класс ResponseWrapperCommentMapperImpl
- добавил структуру таблиц проекта в WiKi проекта

ZhosanDenis commented on src/main/java/ru/skypro/homework/controller/AdController.java:
```diff
@@ -167,8 +167,10 @@ public ResponseEntity<?> deleteAdvertising(@PathVariable long id) {
+            @PathVariable(name = "id") Integer id,
+            @RequestBody Comment comment) {
+        return ResponseEntity.ok(adService.addComment(id, comment));
     }
```
Согласно спецификации RequestBody должно быть объектом типа CreateComment

ZhosanDenis committed:
Изменения от 11.06.2023
- AdController: добавлены вызов методов AdService в эндпоинтах;
- AdEntity: удалена связь с CommentEntity;
- AdService и AdServiceImpl: исправлены методы с комментариями и добавлены методы с объявлениями;
- ResponseWrapperCommentMapperImpl: скорректированы методы;
- UserEntity: удалены связи с AdEntity и CommentEntity.

ZhosanDenis committed:
Merge branch 'dev' into 1-feature-igor

# Conflicts:
#	src/main/java/ru/skypro/homework/controller/AdController.java
#	src/main/java/ru/skypro/homework/repository/CommentRepository.java
#	src/main/java/ru/skypro/homework/repository/UserRepository.java
#	src/main/java/ru/skypro/homework/service/AdService.java
#	src/main/java/ru/skypro/homework/service/impl/AdServiceImpl.java

ZhosanDenis committed:
Изменения от 11.06.2023
- AdRepository: добавлены методы по получению из БД списка объявлений по email пользователя и для проверки наличия объявления по email пользователя
